Resources/translations/pagerfanta.sw.xliff

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="sw" datatype="plaintext" original="file.ext">
+        <body>
+            <trans-unit id="1">
+                <source>previous</source>
+                <target>Nyumba</target>
```

Comment:
I take the precaution of running community-suggested translations through Google translate just in case someone decides to troll us by putting in offensive translations in PRs.  I'm sure your Swahili is better than Google's, so I hope you don't mind me checking, but Google [and Wiktionary](https://en.wiktionary.org/wiki/nyumba) both give the English translation of Nyumba as "House" (and suggest other words for "next").  Please could you just confirm that it is correct?  I'm happy to take your word for it.

Reply:
Thanks, @m453h.  Glad I checked!